linux/LibSensors.c

```diff
@@ -167,6 +171,7 @@ void LibSensors_getCPUTemperatures(CPUData* cpus, unsigned int existingCPUs, uns
    int n = 0;
    for (const sensors_chip_name* chip = sym_sensors_get_detected_chips(NULL, &n); chip; chip = sym_sensors_get_detected_chips(NULL, &n)) {
       const int priority = tempDriverPriority(chip);
+      // printf("chip prefix: %s, path: %s, prio: %d\n", chip->prefix, chip->path, priority);
```

Comment:
No debug code

Reply:
I somehow forgot to remove that line. Sorry about that. Fixed.